src/TestableIO.System.IO.Abstractions.TestingHelpers/MockDirectory.cs

```diff
@@ -39,6 +39,14 @@ public override IDirectoryInfo CreateDirectory(string path)
             return CreateDirectoryInternal(path);
         }
 
+#if FEATURE_FILESYSTEM_UNIXFILEMODE
+        /// <inheritdoc />
+        public override IDirectoryInfo CreateDirectory(string path, UnixFileMode unixCreateMode)
+        {
+            throw new NotImplementedException();
```

Comment:
Should we introduce some helper method for all these `NotImplementedException`s that tells people what is not supported and point them to the issue tracker for requesting it?

Reply:
I replaced all empty `NotImplementedExceptions` in the testinghelpers project with
```csharp
new NotImplementedException(StringResources.Manager.GetString("NOT_IMPLEMENTED_EXCEPTION"));
```

Reply:
Oh, that's nice, I totally forgot about that one 👍